src/main/java/stream/flarebot/flarebot/permissions/PerGuildPermissions.java

```diff
@@ -89,7 +90,8 @@ public boolean isCreator(net.dv8tion.jda.core.entities.User user) {
     }
 
     public boolean isContributor(net.dv8tion.jda.core.entities.User user) {
-        return user.getId().equals("215644829969809421");
+        return user.getId().equals("215644829969809421") || user.getId().equals("203894491784937472") ||
```

Comment:
Who are the people that have been added here?

Reply:
If this includes @AriaHoshiine then Aria shouldn't be classed as an official contributor until at _least_ 5 good quality PRs but preferably 10 good PRs

Reply:
Yeah that was for testing, I assume it's you, Ryan and her.

Reply:
I don't need to be on there because I have `isCreator` and she has not contributed enough to gain that role yet.